test/media/drm_engine_unit.js

```diff
@@ -1087,6 +1088,149 @@ describe('DrmEngine', () => {
           shaka.util.Error.Code.FAILED_TO_GENERATE_LICENSE_REQUEST,
           message, nativeError, undefined));
     });
+
+    // it('should throw a OFFLINE_SESSION_REMOVED error', async () => {
```

Comment:
Because the error makes the createOrLoad throw, I am unable to set up that test (because of the unhandled promise rejection).

I could I set up the test correctly ? 

I've thought to noop createOrLoad in the attach method, because the errors there should have been handled through the onError_method...
```
createOrLoad.catch(() => {});
```
I've tried to noop the catch of allSessionsLoaded_ without success (and it was already nooped in constructor)

Reply:
Turns out it was because I've changed the createOrLoad signature to an async one, errors weren't caught before, so I've silence them